papermill/execute.py

```diff
@@ -229,6 +229,13 @@ def _find_parameters_index(nb):
         raise PapermillException("Multiple parameters tags found")
     return parameters_indices[0]
 
+def _form_escaped_value(str_val):
+    if isinstance(str_val, string_types):
+        str_val = str_val.encode('unicode_escape')
```

Comment:
Does this need to be `string_escape` for Python 2?

Reply:
Nope; see the image above with `unicode_escape` used in python 2. `string_escape` will actually add an extra escape character to the `'` quotes which we don't want either.

Reply:
ah, great, thanks!